src/component/app.jsx

```diff
@@ -167,64 +153,31 @@ export default class App extends Component {
                         </Navigation>
                     </Header>
                     <Drawer className="mdl-color--white">
-                        <span
-                            className={[
-                                styles.drawerTitle,
-                                'mdl-layout-title',
-                            ].join(' ')}
-                        >
+                        <span className={[styles.drawerTitle, 'mdl-layout-title'].join(' ')}>
                             <img
                                 src="public/logo.png"
                                 width="32"
                                 height="32"
                                 className={styles.drawerTitleLogo}
                             />
-                            <span className={styles.drawerTitleText}>
-                                Unleash
-                            </span>
+                            <span className={styles.drawerTitleText}>Unleash</span>
                         </span>
                         <hr />
                         <Navigation className={styles.navigation}>
-                            {createListItem(
-                                '/features',
-                                'Feature Toggles',
-                                'list',
-                                true
-                            )}
-                            {createListItem(
-                                '/strategies',
-                                'Strategies',
-                                'extension',
-                                true
-                            )}
-                            {createListItem(
-                                '/history',
-                                'Event History',
-                                'history',
-                                true
-                            )}
-                            {createListItem(
-                                '/archive',
-                                'Archived Toggles',
-                                'archive',
-                                true
-                            )}
-                            {createListItem(
-                                '/applications',
-                                'Applications',
-                                'apps',
-                                true
-                            )}
+                            {createListItem('/features', 'Feature Toggles', 'list', true)}
```

Comment:
@ivarconr This is the one you pointed out got worse

Reply:
yes, but this could probably be even better by just implementing a `NavigationListItem`-component taking props.